pages/measurement.js

```diff
@@ -152,17 +21,24 @@ export default class Measurement extends React.Component {
     if (query.input) {
       params['input'] = query.input
     }
-    let msmtResult = await client.get('/api/v1/measurements', {
-      params
-    })
-
+    let [msmtResult, countriesR] = await Promise.all([
+      client.get('/api/v1/measurements', {
+        params
+      }),
+      client.get('/api/_/countries')
+    ])
```

Comment:
I don't remember what happens if one of the two promises fails and what is the correct pattern for handling this, but this is probably something to think about.

Reply:
There is a [this pattern](https://davidwalsh.name/promises-results) that I just came across. Maybe a utility function to keep it readable? Like this:

Reply:
Actually I think we are better of failing if one of the promises fail as it will likely break the UI. Maybe we should do something with the event like log it to the error log.